website/source/guides/packer-on-cicd/building-virtualbox-image.html.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: guides
+sidebar_current: guides-packer-on-cicd-build-virtualbox
+page_title: Building a VirtualBox Image with Packer in TeamCity
+description: |-
+  ...
+---
+
+# Building a VirtualBox Image with Packer in TeamCity
+
+This guide walks through the process of building a VirtualBox image using Packer on a new TeamCity Agent. Before getting started you should have access to a TeamCity Server.
+
+The Packer VirtualBox builder requires access to VirtualBox which should run on a bare-metal machine as virtual machines should not run inside other virtual machines. This is also true for the [VMWare](https://www.packer.io/docs/builders/vmware.html) and the [QEMU](https://www.packer.io/docs/builders/qemu.html) Packer builders.
```

Comment:
> as virtual machines should not run inside other virtual machines

I think it's more accurate to say that virtualization is generally not supported on cloud instances.

Maybe

> The Packer VirtualBox builder requires access to VirtualBox, which generally needs to run on a bare-metal machine.